recipes/express/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: express
+  version: "1.5.1"
+
+build:
+  number: 0
+
+source:
+  fn: express_1.5.1_linux_x64.tar.gz # [linux64]
+  url: https://depot.galaxyproject.org/software/express/express_1.5.1_linux_x64.tar.gz # [linux64]
+  fn: express_1.5.1_darwin_x64.tar.gz # [osx]
+  url: https://depot.galaxyproject.org/software/express/express_1.5.1_darwin_x64.tar.gz # [osx]
```

Comment:
On the other hand we also compile them on such an old system in Galaxy. So not sure if this is required.